apps/aecore/src/aec_governance.erl

```diff
@@ -158,7 +158,8 @@ tx_base_gas(name_update_tx, _)            -> ?TX_BASE_GAS;
 tx_base_gas(oracle_extend_tx, _)          -> ?TX_BASE_GAS;
 tx_base_gas(oracle_query_tx, _)           -> ?TX_BASE_GAS;
 tx_base_gas(oracle_register_tx, _)        -> ?TX_BASE_GAS;
-tx_base_gas(oracle_response_tx, _)        -> ?TX_BASE_GAS.
+tx_base_gas(oracle_response_tx, _)        -> ?TX_BASE_GAS;
+tx_base_gas(paying_for_tx, _)             -> ?TX_BASE_GAS div 5.
```

Comment:
Somewhere we should record the ratio behind this 1/5th.

Reply:
How about here: https://github.com/aeternity/protocol/pull/443 (you have already approved it!)

Reply:
There should be an inline comment as well.